### PR TITLE
Add entry castelli2021study

### PR DESCRIPTION
**Pull request by gfcas**

### CODE REVIEW

File: publications-2021.bib [hunk 1]
```diff
@@ -67,4 +67,18 @@ @article{Ross2021
   journal = {Frontiers in Physiology}
 }
 
+@InProceedings{castelli2021study,
+  author    = {G. F. Castelli, W. D{\"o}rfler},
```
Comment (bangerth) [inline publications-2021.bib:71]: bibtex will want you to write this as
```suggestion
  author    = {G. F. Castelli and W. D{\"o}rfler},
```

File: publications-2021.bib [hunk 2]
```diff
@@ -67,4 +67,18 @@ @article{Ross2021
...
+  title     = {Study on an Adaptive Finite Element Solver for the {Cahn--Hilliard} Equation},
+  editor    = {F. J. Vermolen, C. Vuik},
```
Comment (bangerth) [inline publications-2021.bib:73]:
```suggestion
  editor    = {F. J. Vermolen and C. Vuik},
```

**Review by bangerth — COMMENTED** [review]
Nice! Would you mind making the two suggested changes?

**Comment by gfcas** [comment]
Ups, there I copied to fast from plain text. Corrected :-)

**Review by bangerth — APPROVED** [review]

**Comment by bangerth** [comment]
Thanks!

**Comment by gfcas** [comment]
Sure, thank you. Furthers will follow.

**Comment by bangerth** [comment]
We're looking forward to them!

**Pull request merged**